hooks/post/30-upgrade.rb

```diff
@@ -66,6 +66,15 @@ def remove_gutterball
   end
 end
 
+def remove_event_queue
+  queue_present = `qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671 | grep :event | wc -l`.chomp.to_i
```

Comment:
@ehelms @ekohl Right they can change I did not think of that if qpid ever had a massive change or we changed where the certs are stored. Did you want me to pull the cert location from the answer file? I can do a grep of that and pull it from there if needed.